implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -56,16 +56,10 @@
     private final InternalLogger logger = InternalLoggerFactory.getInstance(DynatraceExporterV2.class);
     private static final Map<String, String> staticDimensions = Collections.singletonMap("dt.metrics.source", "micrometer");
 
-    private final String endpoint;
-    private final boolean ignoreToken;
     private final MetricBuilderFactory metricBuilderFactory;
 
     public DynatraceExporterV2(DynatraceConfig config, Clock clock, HttpSender httpClient) {
         super(config, clock, httpClient);
-        this.endpoint = config.uri();
-        showErrorIfEndpointIsInvalid(endpoint);
-        ignoreToken = shouldIgnoreToken(config);
-        logger.info("Exporting to endpoint {}", this.endpoint);
```

Comment:
Do we still have an info-level log for the first endpoint value and each time it changes?